packages/perf-benchmarks/src/__benchmarks__/engine-dom/benchmark-slot-ss/ss-slot-create-container.benchmark.js

```diff
@@ -7,9 +7,9 @@
 
 import { createElement } from 'lwc';
 
-import SlotUsage from 'benchmark/slotUsageComponent';
-import Store from 'benchmark/store';
-import { insertComponent, destroyComponent } from 'benchmark/utils';
+import SlotUsage from 'perf-benchmarks-components/dist/dom/benchmark/slotUsageComponent/slotUsageComponent.js';
+import Store from 'perf-benchmarks-components/dist/dom/benchmark/store/store.js';
```

Comment:
We can just have one big `dist/index.js` file referenced by `package.json` that contains all the exported components. But then benchmarks that only need component X will still get served component Y. This won't technically affect the benchmark numbers, since it impacts JS parse/compile time and we're measuring runtime, but it would make the tests run slower overall.

Another alternative: use Node.js [export maps](https://medium.com/swlh/npm-new-package-json-exports-field-1a7d1f489ccf).

Reply:
Ah too bad, looks like Rollup understands Node's new `"exports"` field, but the Tachometer server does not. For now it seems to me that the `dist` solution is best.